package/MDAnalysis/analysis/pca.py

```diff
@@ -273,6 +309,38 @@ def _conclude(self):
         self._calculated = True
         self.n_components = self._n_components
 
+    @property
+    def p_components(self):
+        wmsg = ("The `p_components` attribute was deprecated in "
```

Comment:
We'll need tests for these new warnings.

Reply:
Tests should be there

Reply:
Sorry my bad, my eyes completely glossed over `test_pca_attr_warning` for some reason.